shared/chat/conversation/create-team-header/container.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import CreateTeamHeader from '.'
+import {connect} from 'react-redux'
+import {navigateAppend} from '../../../actions/route-tree'
+
+export type DispatchProps = {|
```

Comment:
why export this type? also what matches this type? not the mapDispatchToProps below?